Anomaly/Patches/ThingDefs_Races/Races_Entities_Misc.xml

```diff
@@ -62,9 +62,9 @@
 	<Operation Class="PatchOperationAdd">
 		<xpath>Defs/ThingDef[defName="Nociosphere"]/statBases</xpath>
 		<value>
-			<MeleeDodgeChance>1</MeleeDodgeChance>
-			<MeleeCritChance>0.5</MeleeCritChance>
-			<MeleeParryChance>0</MeleeParryChance>
+			<MeleeDodgeChance>0.03</MeleeDodgeChance>
+			<MeleeCritChance>0.26</MeleeCritChance>
+			<MeleeParryChance>0.57</MeleeParryChance>
```

Comment:
High parry doesn't make much sense to me, not least because the spheres don't have any apparent means to parry melee attacks. Given how fast they teleport around when active, I'm alright with the spheres not being super strong against melee since they're going to be hard to get to in the first place.

Reply:
Can set to minimal or even 0 parry chance, went with the spreadsheet values for consistency's sake. But who knows, they are made out of multiple moving plates, maybe they *can* in fact parry incoming attacks.

Reply:
what melee attack would this thing even need to parry, a light saber?